versions/wmde9/built_versions.log

```diff
@@ -0,0 +1,4 @@
+MediaWiki 1.36
```

Comment:
is this automatically generated file? Should be 1.37 no?

Reply:
automatically generated. Wrong value stemmed from incorrect RELEASE_MAJOR_VERSION env variable: https://github.com/wmde/wikibase-release-pipeline/pull/376